src/CounterInterceptor/MyWorkflow.workflow.cs

```diff
@@ -0,0 +1,56 @@
+namespace TemporalioSamples.CounterInterceptor;
+
+using Temporalio.Workflows;
+
+[Workflow]
+public class MyWorkflow
+{
+    private string name = string.Empty;
+    private string title = string.Empty;
+    private bool exit; // automatically defaults to false
+
+    [WorkflowRun]
+    public async Task<string> ExecAsync()
+    {
+        // wait for greeting info
+        await Workflow.WaitConditionAsync(() => name != null && title != null);
```

Comment:
This condition always passes

Reply:
I'm not sure I understand what you mean. Can you clarify please?

Reply:
Name and title are never null, so what is the purpose of this condition?